src/PostProcesses/postProcess.ts

```diff
@@ -677,17 +732,32 @@ export class PostProcess {
 
     private _disposeTextures() {
         if (this._shareOutputWithPostProcess || this._forcedOutputTexture) {
+            this._disposeTextureCache();

```

Comment:
Was it a bug of the previous code to not release the textures in this case?

Reply:
I think in case `this._shareOutputWithPostProcess` or `this._forcedOutputTexture` was defined, textures in the texture array are released in the setter. Anyhow, defensively this makes sense.